### ISSUE TITLE
Problemi con funzione menu

### ISSUE THREAD

**Issue by roberbi** [PorteAperteSulWeb/pasw2015#137]
Ho aggiornato passw15 alla versione 1.6.6 e ora ho problemi con la gestione dei menu dalla bacheca di wordpress, non funziona. È normale che devo utilizzare il personalizzatore del tema? Questo comportamento è corretto, oppure ho dei problemi da risolvere con la mia versione di wordpress 4.3?

**Comment by milesimarco** [comment]
No, non è normale, almeno per il momento. La direzione della comunità di WordPress è comunque quella di unificare tutto ciò che riguarda "Aspetto" all'interno del personalizzatore.

Per caso stai lavorando su un'installazione recente? Se dalla bacheca clicchi Aspetto > Menù dove arrivi?

**Comment by roberbi** [comment]
Grazie 
Sto lavorando sulla versione wordpress 4.3. Quando clicco su menu non arrivo da nessuna parte, gira a vuoto. A volte riesco ad entrare nella scheda dei menu però non si riesce ad accedere alle funzioni.
Roberto sito www.iisprever.gov.it

Colgo l'occasione per ringrazierti dell'ottimo lavoro che svolgi per tutti noi
